.travis.yml

```diff
@@ -2,9 +2,18 @@ language: python
 python:
     - "3.5"
     - "3.4"
+sudo: required
+before_script:
+    - psql -c "create database sdx;" -U postgres
+    - psql -c "create user sdx with password 'sdx';" -U postgres
+    - psql -c "grant all privileges on database sdx to sdx;" -U postgres
+    - psql -d sdx -a -f schema.sql -U sdx
 install: 
-- pip install -r requirements.txt
-- pip install -r test_requirements.txt
+    - pip install -r requirements.txt
+    - pip install -r test_requirements.txt
 script: 
-- flake8
-- python -m unittest tests/*.py
+    - make test
+services:
+    - postgresql
+addons:
+   postgresql: "9.4"
```

Comment:
Think we had a similar issue before. 9.6 should be available (https://docs.travis-ci.com/user/database-setup/#PostgreSQL) but you may have to specify the Linux container.